x-pack/plugins/ml/public/application/routing/breadcrumbs.ts

```diff
@@ -13,118 +13,135 @@ import { ChromeBreadcrumb } from '@kbn/core/public';
 
 import { NavigateToPath } from '../contexts/kibana';
 
-export const ML_BREADCRUMB: ChromeBreadcrumb = Object.freeze({
+export type MlBreadcrumb = ChromeBreadcrumb & {
+  /**
+   * Root level navigational breadcrumbs shouldn't be set in serverless, as they are
+   * controlled by each project's navigation tree
+   */
+  hiddenInServerless?: true;
```

Comment:
We try to avoid adding logic which mentions serverless outside of the initial app initialization. 
Instead we name things based on what the are specifically doing.
This variable could be called `hideRootLevelNav` or something like that.

Reply:
Thanks! will improve